micro_dl/inference/image_inference.py

```diff
@@ -722,11 +785,12 @@ def predict_2d(self, chan_slice_meta):
                         self.crop_shape,
                         self.image_format,
                     )
+                
+                #pass to network for prediction
```

Comment:
Some more nitpicking: for proper pep you need a space between your pound symbol and your comment.